lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/persistence/DBOStorageLocation.java

```diff
@@ -208,6 +209,18 @@ public String toString() {
 			
 			@Override
 			public DBOStorageLocation createDatabaseObjectFromBackup(DBOStorageLocation backup) {
+				// PLFM-5769 - Remove after migrated
```

Comment:
Existing external S3 Storage Locations will be modified to NOT end with one (or more) trailing slashes.